packages/sdk/src/internal/data-service-api/data-service-dialects-api.ts

```diff
@@ -66,13 +66,13 @@ export class DataServiceDialectsApiClient implements DataServiceDialectsApi {
     );
   }
 
-  async findAll(query?: FindDialectQuery): Promise<DialectAccountDto[]> {
+  async findAll(query?: FindDialectQuery, limit?: number): Promise<DialectAccountDto[]> {
     const token = await this.tokenProvider.get();
     return withReThrowingDataServiceError(
       axios
         .get<DialectAccountDto[]>(`${this.baseUrl}/api/v1/dialects`, {
           headers: createHeaders(token),
-          ...(query && { params: query }),
+          ...(query && { params: query, limit }),
```

Comment:
Same comment here as in data-service pr, should the limit be part of query params?

Reply:
Yeah let me fix these up.